aptos-move/aptos-vm/src/move_vm_ext/session/user_transaction_sessions/user.rs

```diff
@@ -104,19 +104,16 @@ impl<'r> UserSession<'r> {
 
             self.session.execute(|session| {
                 let module_id = module.self_id();
-                let init_function_exists = staging_module_storage
-                    .load_function(&module_id, init_func_name, &[])
-                    .is_ok();
+                let init_func_result =
+                    staging_module_storage.load_function(&module_id, init_func_name, &[]);
```

Comment:
I think it may be even clearer without the intermediate variable.

Reply:
Removed intermediate variable